src/provider.h

```diff
@@ -18,7 +18,7 @@
 #include <openssl/proverr.h>
 #include <openssl/core_names.h>
 
-#define UNUSED __attribute__((unused))
+#define UNUSED __attribute__((__unused__))
```

Comment:
Why this change ?

Reply:
There was issue in the past with the standard `noreturn` (from `#include <stdnoreturn.h>`) which mangled system headers that use `__attribute__((noreturn))` and mangled it to `__attribute__((_Noreturn))` which in turn `clang` ignored, so it's kind of habit to use the underscored variants of attributes, especially in headers.

There's no other reason here, so this could be dropped, I did this on autopilot.

Reply:
Removed.

Reply:
As lon as I know the reason I am fin with it